cmd/server/assets/login/reset-password.html

```diff
@@ -70,8 +70,12 @@
           flash.alert('Password reset email sent.');
         }).catch(function(error) {
           flash.clear();
-          flash.error(error.message);
-          $submit.prop('disabled', false);
+          if (error.code = "auth/user-not-found") {
```

Comment:
So... this fixes the issue, but it doesn't actually stop anyone with a Web Inspector. You can still see the response sent.

Reply:
we should always say something like - "check your email for your reset password link" and leave it at that.

Reply:
I mean the response from calling the firebase API - until this is all server side, someone can just look at what firebase is returning. Or they could just talk to firebase directly with our API key.

Reply:
This is what I was talking about in the description. We need an SMTP server to send mail if we want to do it ourselves or we could wrap it all in a node.js cloud function (gross).

Reply:
I mean. The other option is to bundle the node.js runtime into our container and exec...